src/components/InviteLearnersModal/index.jsx

```diff
@@ -12,7 +13,6 @@ import {
   validateEmailAddresses,
```

Comment:
Is there a future ticket to be filed to move these validators to frontend-platform?

Reply:
Good question, I think it's something we can talk about with @davidjoy and #fed-ex. I actually just had a conversation with people today about what kinds of utility functions belong there. I think for validators like this, I lean towards not rolling our own and moving to a third-party library. (And for forms in general, because we sure do have a lot of forms, and there are some nice libraries out there to handle stuff for us. Also just had a conversation today about formik...)